### PR TITLE
zabbix update

### PR DESCRIPTION
**Pull request by Canpy**
zabbix.sls updated

### CODE REVIEW

**Comment by rosscdh** [comment]
delete BIN +3.7 MB [bit_base_minions/files/chocolatey.1.0.0.nupkg](https://github.com/rosscdh/bit_base_minions-formula/pull/1/files#diff-c3b90d7395a707b5c6b1d8e330949e06757c17fb992713f0d4e1b3e546a8749f)

File: bit_base_minions/windows/chocolatey.sls [hunk 1]
```diff
@@ -9,9 +9,34 @@
 #
 # chocolatey broke their installation
 #
+#bit_base_minions_install_chocolatey:
+  #cmd.run:
```
Comment (rosscdh) [inline bit_base_minions/windows/chocolatey.sls]: delete this commented out stuff

File: bit_base_minions/windows/zabbix.sls [hunk 2]
```diff
@@ -53,3 +53,22 @@ windows-retsart-zabbix-agent:
       - file: {{ psk_file }}
       - windows_zabbix_agent_conf
 
+
+
+
+           
+{%- if '_connection_url' in settings.zabbix %}
+update_zabbix:
```
Comment (rosscdh) [inline bit_base_minions/windows/zabbix.sls]: add the same to the linux one.. or rather a shared zabbix_update.sls that is included in windows and linux from the init.sls

**Comment by rosscdh** [comment]
and ignore all of reports/*

**Review by rosscdh — CHANGES_REQUESTED** [review]
.gitignore reports/*
dlete the files chocolty 1.0.0.nupkg
and the other comments inline

File: bit_base_minions/windows/zabbix.sls (hunk 2 above)
Comment (Canpy) [reply bit_base_minions/windows/zabbix.sls:61]: delete line 60 AND 61? or just 60 of zabbix.sls

Comment (Canpy) [reply bit_base_minions/windows/zabbix.sls]: dont know exactly what you mean with that

**Review by rosscdh — APPROVED** [review]

**Pull request merged**